index.html

```diff
@@ -684,31 +743,30 @@
               </div>
               <div class="card-content">
                 <div class="image">
-                  <img src="./img/ev-6.jpeg">
+                  <img alt="alt text" src="./img/ev-6.jpeg">
                 </div>
               </div>
               <div class="card-footer">
-                <a class="card-footer-item">Preview</a>
-                <a class="card-footer-item">Source Code</a>
+                <a tabindex="0" tabindex="0" class="card-footer-item">Preview</a>
+                <a tabindex="0" tabindex="0" class="card-footer-item">Source Code</a>
               </div>
             </div>
           </div>
         </div>
       </div>
     </div>
-    </div>
-  </div>
+  </section>
 
-  <div class="section">
+  <section id="forum" class="section">
     <div class="container">
       <div class="section-heading has-text-centered">
-        <div class="title is-2">Forum</div>
-        <div class="subtitle is-5">Discuss any topic with no leaving this page</div>
+        <h2 class="title is-2">Forum</h2>
+        <h5 class="subtitle is-5">Discuss any topic with no leaving this page</h5>
       </div>
 
       <div class="columns">
-        <div class="column is-3">
-          <a class="button is-primary is-block is-alt is-large">New Post</a>
+        <h3 class="column is-3">
+          <a tabindex="0" tabindex="0" class="button is-primary is-block is-alt is-large">New Post</a>
           <div class="menu">
             <ul class="menu-list forum-list">
```

Comment:
Items inside the list should probably be links as well as they are the forum's tags.

Reply:
Done, I changed `span` with `a`